libs/api/domains/license-service/src/lib/graphql/payload.model.ts

```diff
@@ -45,4 +45,16 @@ export class Payload {
   @IsObject()
   // eslint-disable-next-line @typescript-eslint/ban-types
   rawData?: object
+
+  @Field(() => String)
+  licenseNumber?: string
+
+  @Field(() => Boolean, { nullable: true })
+  expired?: boolean | null
+
+  @Field(() => String)
+  title?: string
+
+  @Field(() => String)
+  logo?: string
```

Comment:
Should this be on `GenericLicense` since it's the same for all license? Or is it unique for the user?